framework/db/mysql/Schema.php

```diff
@@ -53,6 +68,128 @@ class Schema extends \yii\db\Schema
         'varbinary' => self::TYPE_BINARY,
     ];
 
+    /**
+     * @inheritDoc
+     */
+    protected function resolveTableName($name)
+    {
+        $resolvedName = new TableSchema();
+        $parts = explode('.', str_replace('`', '', $name));
```

Comment:
Maybe in a separate PR. If current PR doesn't support ANSI quotes and noone complains then probably it's fine to have it as is.